README.md

```diff
@@ -17,6 +17,5 @@ PLAY_VERSION=2.X sbt "project itServer" run
 ## Installation
 
 ```sbt
-// replace suffix play-26 with play-25 if using with Play 2.5
 libraryDependencies += "uk.gov.hmrc" %% "play-nunjucks-viewmodel" % "x.y.z-play-26"
```

Comment:
To confirm, we're not supporting Play 2.7 at the moment? I'm fine with that, just wanting to confirm.

Reply:
Correct, that's a bigger piece.